### ISSUE TITLE
a11y: Consider light/dark color schemes

### ISSUE THREAD

**Issue by twhite313** [paceaux/selector-finder#46]
The teal color is difficult to read on light themed terminal. 
Consider alternate palette. 

(No, we don't all use dark mode. :)  )

<img width="922" height="563" alt="Image" src="https://github.com/user-attachments/assets/2935f0fa-479a-45b7-81f5-0cd0bff581f2" />

**Comment by paceaux** [comment]
thanks for that feedback! 

Do you think a basic black/white color scheme would work better, @twhite313 ?

**Comment by twhite313** [comment]
Basic black/white will certainly work. 
If you want to keep your colors, you can - the teal just needs to be darker when it's on white. Something like #158484

(I'm not sure the exact colors you are using. If you provide them I can help come up with accessible contrast for you)

**Comment by paceaux** [comment]
@twhite313 The color I'm using is cyan https://github.com/paceaux/selector-finder/blob/develop/src/logger.js#L95

TBH I feel awful not considering the accessibility _in the console_. I'll do some tests to see what will be a good color contrast.

**Comment by paceaux** [comment]
Did some checking. Going to use `#0B6060`. this passes WCAG AA and AAA accessibility. 

Looks more green than blue which is fine b/c it's SelectorHound. Hounds like grass. 

Looks readable in dark and light terminals

<img width="587" height="463" alt="Image" src="https://github.com/user-attachments/assets/40a11b57-6f09-4fca-90af-681699349330" />

<img width="577" height="383" alt="Image" src="https://github.com/user-attachments/assets/55c47c0b-2442-48ad-884e-b6d069c07d4b" />

**Comment by paceaux** [comment]
Need to find out why I have one pipe consistently breaking. Once I get to that, will merge in

**Comment by twhite313** [comment]
> [@twhite313](https://github.com/twhite313) The color I'm using is cyan https://github.com/paceaux/selector-finder/blob/develop/src/logger.js#L95
> 
> TBH I feel awful not considering the accessibility _in the console_. I'll do some tests to see what will be a good color contrast.

Yeah, I'm one of those weirdos who doesn't prefer dark mode everywhere. :) 
And oddly my terminal spits out both modes for some reason. No idea why and too lazy to troubleshoot the settings. 

But also, I run terminal in VS Code, where I have a Sepia theme... so...

**Comment by paceaux** [comment]
Turns out the clock emoji has a long history and it uses two unicode sequences; one for emoji and another for text presentation. It has dual presentations depending on mode and apparently b/c VS Code is a fancy web browser under the hood, it handles the clock emoji as two spaces, while a terminal handles it as one. Therefore causing the break. 

So gonna switch emojis.

**Comment by paceaux** [comment]
@twhite313 Not necessary but if you wanted to review the changes you're welcome to look at the pull request
